invenio_access/upgrades/access_2015_05_06_accROLE_accACTION_accARGUMENT_id.py

```diff
@@ -70,7 +70,9 @@ def do_upgrade():
         CHANGE COLUMN "id_accARGUMENT" "id_accARGUMENT" INT(15) NULL ,
         CHANGE COLUMN "argumentlistid" "argumentlistid" MEDIUMINT(8) NULL ,
         ADD COLUMN "id" INT(15) UNSIGNED NOT NULL AUTO_INCREMENT,
-        DROP PRIMARY KEY,
+        DROP KEY id_accROLE,
```

Comment:
We should keep the individual indexes on `id_acc*`, but probably we need to call the `DROP PRIMARY KEY` separately and catch an exception.